PostHog/PostHogConfig.swift

```diff
@@ -7,6 +7,20 @@
 import Foundation
 
 @objc(PostHogConfig) public class PostHogConfig: NSObject {
+    enum Defaults {
+        #if os(tvOS)
+            static let flushAt: Int = 5
+            static let maxQueueSize: Int = 100
+            static let maxBatchSize: Int = 5
+            static let flushIntervalSeconds: TimeInterval = 5
```

Comment:
i think these 2 is fine as it is
the bigger the batch the better
the one is is a trade off of battery and resources, but we don't need more than 30s I think